superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -89,10 +89,12 @@ const StyledHeader = styled.div`
   z-index: 2;
 `;
 
-const StyledContent = styled.div`
+const StyledContent = styled.div<{
+  fullSizeChartId: number | null;
+}>`
   grid-column: 2;
   grid-row: 2;
-  z-index: 1;
+  z-index: ${({ fullSizeChartId }) => (fullSizeChartId ? 1000 : 1)};
```

Comment:
I think `z-index: 2` also enough, I think better not to use big zIndexes because if need later harder override them

Reply:
I'd like to make a place to store z-index values as constants so that they are all collected in one place and can be adjusted and compared easily. Maybe this PR isn't the place to do that though.

Reply:
I agreed. There seems to be no place to manage all z-index values.